packages/rich-text-structure/src/get-active-format.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+
+import { find } from 'lodash';
+
+export function getActiveFormat( { formats, start }, formatType ) {
```

Comment:
specifying a `formatType` here seems a bit unexpected. does this not return format of all types?

Reply:
In all use cases so far, I've needed information about the current format by type only. I've never had a case where I needed _all_ active formats. We could return all formats if `formatType` is `undefined` if you like, but I usually just add functionality as uses cases pop up.

Reply:
> We could return…as uses cases pop up.

that's not where I was going with it. I'd say I find this function name confusing because if I know the type I'm looking for I'd expect to call something like `isFormatActive( record, type )` but if I wanted to know the active format for a range I'd expect to call `getActiveFormats( record )`

more or less still confused what the purpose is here of this function